ramalama/cli.py

```diff
@@ -956,6 +956,16 @@ def serve_parser(subparsers):
     parser.add_argument(
         "--rag", help="RAG vector database or OCI Image to be served with the model", completer=local_models
     )
+    parser.add_argument("--model_draft", help="Draft model", completer=local_models)
+    parser.add_argument(
```

Comment:
--ngld option is too much I think...

How many users will want a different --ngl value to --ngld value? Maybe 0.00001% will actually use this option.

I think just a `--model_draft` option is good for this PR and just use the --ngl value. We have to be careful of maintenance of the codebase.

We do not want as many options as llama-server .

Reply:
ack

Reply:
It is possible you want to have higher ngld when you are dealing with big model which does not fully fits into VRAM.

Reply:
Would it be better to either default it to 999 or remove it from here ?